### PR TITLE
fix(translation): Fix a few translations

### PR DESCRIPTION
**Pull request by lynnagara**

### CODE REVIEW

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">You should update CHANGES due to the size of this PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Comment by MaxBittker** [comment]
there's some really good stuff in here, i need to double check the platform name stuff I think, so gonna come back tomorrow

File: src/sentry/static/sentry/app/views/onboarding/project/platformpicker.jsx [hunk 1]
```diff
@@ -87,24 +88,24 @@ class PlatformPicker extends React.Component {
...
                 className="platform-filter"
-                label="Filter"
+                label={t('Filter')}
                 placeholder="Filter"
                 onChange={e => this.setState({filter: e.target.value})}
               />
             </div>
           </li>
-          {categoryList.map(categoryName => {
+          {allCategories.map(({id, name}) => {
```
Comment (MaxBittker) [inline src/sentry/static/sentry/app/views/onboarding/project/platformpicker.jsx]: nice job detangling this logic  - I'm slightly afraid that a problem will show up when the strings get translated, but it looks like you handled the cases correctly as far as i can see

File: src/sentry/static/sentry/app/views/projectInstall/overview.jsx [hunk 2]
```diff
@@ -92,7 +92,7 @@ class ProjectInstallOverview extends React.Component {
           {tct(
             `
              For a complete list of
-             client integrations, please visit see [docLink:our in-depth documentation].
+             client integrations, please see [docLink:our in-depth documentation].
```
Comment (MaxBittker) [inline src/sentry/static/sentry/app/views/projectInstall/overview.jsx]: 🙌

File: src/sentry/static/sentry/app/views/organizationDashboard.jsx [hunk 3]
```diff
@@ -192,14 +192,14 @@ const ProjectList = createReactClass({
...
+          {t('View All')}
         </Link>
-        <h6 className="nav-header">Projects</h6>
+        <h6 className="nav-header">{t('Projects')}</h6>
         {bookmarkedProjects.length === 0 && (
           <div className="alert alert-info" style={{marginBottom: 10}}>
-            Bookmark your most used{' '}
-            <Link to={`/organizations/${org.slug}/teams/`}>projects</Link> to have them
-            appear here.
+            {tct('Bookmark your most used [projects:projects] to have them appear here', {
```
Comment (MaxBittker) [inline src/sentry/static/sentry/app/views/organizationDashboard.jsx]: `[projects:projects] ` what is this syntax? does `[projects]` work?

**Review by MaxBittker — APPROVED** [review]
image

**Comment by MaxBittker** [comment]
actually, check out percy before you merge this, it found a missing period (maybe intentional?) and a messed up storybook example.

**Comment by lynnagara** [comment]
Ah, that was intentional. We don't normally have periods at the end of those warning messages so thought I'd fix it for some consistency since I was adding translations there

File: src/sentry/static/sentry/app/views/organizationDashboard.jsx (hunk 3 above)
Comment (lynnagara) [reply src/sentry/static/sentry/app/views/organizationDashboard.jsx]: Hmm, yeah i think it should be `[projects]`

**Comment by lynnagara** [comment]
Unfortunately I don't think we can fix the storybook thing... doesn't look so good but that's just how it works?

**Pull request merged**